### DISCUSSION TITLE
Guitar removal

### DISCUSSION THREAD

**Discussion by Darkil1** [Anjok07/ultimatevocalremovergui#1610] [Q&A] [answered]
Hello, is it possible in your application to cut only the guitar from a song without affecting other instruments?

**Comment by damiensellier** [comment]
The only way I got guitars parts separated from the beat and vocals was using MDX NET Inst-Main  and "Kuielab_a/b other" algo. But there was the keyboards laced with it.

If you search for "guitar" in : https://docs.google.com/document/d/17fjNvJzj8ZGSer7c7OFe_CNfUKbAxEh_OBv94ZdRG5c/edit?pli=1&tab=t.0#heading=h.roiuj54hzww3   you won't find anything which is a bad sign :(

**Comment by SkyKuker** [comment, marked as answer]
Hi!

I've had quite an impressive experience (_YMMV_) with the `v4 | htdemucs_6s` model. Select "Guitar" in the stem drop-down list, and you're good to go.

Hope that helps you out!

P.S. This might be the only model that separates guitars in the entire utility.

Edit: spelling.
